### PR TITLE
added check for case id

### PR DESCRIPTION
**Pull request by EmmaCAndersson**
# Adding case id check for sample sheet

This PR contains an additional feature in bin/check_samplesgeet.py to check that a case id for each sample is present in the sample sheet. Some code for checking that the column name case_id is present was already in the script.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test_one_sample,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: bin/check_samplesheet.py [hunk 1]
```diff
@@ -35,6 +35,7 @@ def __init__(
         first_col="fastq_1",
         second_col="fastq_2",
         single_col="single_end",
+        case_id_col="case_id"
```
Comment (jemten) [inline bin/check_samplesheet.py:38]:
```suggestion
        case_id_col="case_id",
```

Comment (jemten) [reply bin/check_samplesheet.py:38]: Sneaky, sneaky comma 🥷

Comment (EmmaCAndersson) [reply bin/check_samplesheet.py]: thanks!

**Review by jemten — COMMENTED** [review]
Looks good. @EmmaCAndersson could you update your branch with the latest changes from the raredisease dev branch?

**Comment by ramprasadn** [comment]
Nice work @EmmaCAndersson! Could you also update CHANGELOG.md?

**Comment by EmmaCAndersson** [comment]
> Nice work @EmmaCAndersson! Could you also update CHANGELOG.md?

Yes! That's done now.

**Review by ramprasadn — APPROVED** [review]
LGTM! 🚀

**Pull request merged**